packages/virtualized-lists/Lists/VirtualizedList.js

```diff
@@ -21,15 +24,15 @@ import type {
   Separators,
 } from './VirtualizedListProps';
 
-import RefreshControl from '../Components/RefreshControl/RefreshControl';
-import ScrollView from '../Components/ScrollView/ScrollView';
-import View from '../Components/View/View';
-import Batchinator from '../Interaction/Batchinator';
-import {findNodeHandle} from '../ReactNative/RendererProxy';
-import flattenStyle from '../StyleSheet/flattenStyle';
-import StyleSheet from '../StyleSheet/StyleSheet';
-import clamp from '../Utilities/clamp';
-import infoLog from '../Utilities/infoLog';
+import RefreshControl from 'react-native/Libraries/Components/RefreshControl/RefreshControl';
+import ScrollView from 'react-native/Libraries/Components/ScrollView/ScrollView';
+import View from 'react-native/Libraries/Components/View/View';
+import Batchinator from 'react-native/Libraries/Interaction/Batchinator';
```

Comment:
This part is a little problematic, since these APIs are private. It means the VirtualizedList package is silently coupled to the version of RN.

Reply:
I had a forked package of VirtualizedList before where I copied Batchinator, etc to be local to the package. The full set of transitive dependencies is manageable. But it means duplicates in the bundle which does have a size impact.

Reply:
For View, ScrollView, etc we can import them from the react-native public entry-point though.

I.e.
```js
import {RefreshControl, ...} from 'react-native';
```

Reply:
I've just updated the `react-native` imports to use public entry-points